lib/active_record/connection_adapters/makara_abstract_adapter.rb

```diff
@@ -107,10 +107,10 @@ def custom_error_message?(connection, message)
 
 
       hijack_method :execute, :exec_query, :exec_no_cache, :exec_cache, :transaction
-      send_to_all :connect, :reconnect!, :verify!, :clear_cache!, :reset!, :pool=
+      send_to_all :connect, :reconnect!, :verify!, :clear_cache!, :reset!
 
-      control_method :close, :steal!, :expire, :lease, :in_use?, :owner, :schema_cache,
-         :schema_cache=, :_run_checkin_callbacks, :_run_checkout_callbacks, :lock, :seconds_idle
+      control_method :close, :steal!, :expire, :lease, :in_use?, :owner, :schema_cache, :pool=, :pool,
```

Comment:
might be worth documenting `send_to_all` vs `control_method` and the rationale here